signer.gemspec

```diff
@@ -19,5 +19,9 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "rake"
   gem.add_development_dependency "rspec"
 
-  gem.add_runtime_dependency "nokogiri", ">= 1.5.1"
+  if RUBY_VERSION >= '2.1'
+    gem.add_runtime_dependency "nokogiri", ">= 1.5.1"
+  else
+    gem.add_runtime_dependency "nokogiri", "~>1.6.8"
+  end
```

Comment:
This won't work, because gemspec is evaluated when building the `.gem`. Let's drop `2.0`. Can you add `spec.required_ruby_version = '>= 2.1.0'` and remove `2.0` from travis.yml?